src/components/CreateCertificate/CreateCertificate.tsx

```diff
@@ -1,237 +1,195 @@
+        <DialogTitle>Certification Successful</DialogTitle>
+        <DialogContent>
+          <span>
+            View your certification broadcasted on-chain&nbsp;
+            <a target="_blank" rel="noreferrer" href={`https://preprod.cardanoscan.io/transaction/${transactionId}`}>here</a>!
+          </span>
+        </DialogContent>
+      </Dialog>
+    );
+  }
+  
+  const MetadataModal = () => {
     return (
+      <Dialog open={openMetadataModal} onClose={onCloseMetadataForm} fullWidth maxWidth="md">
+        <DialogTitle>Certification Metadata</DialogTitle>
+        <DialogContent dividers>
+          <CertificationMetadataForm
+            uuid={uuid}
+            isReviewCertification={true}
+            onClose={onCloseMetadataForm}
+          />
+        </DialogContent>
+      </Dialog>
+    );
+  }
+  
+  return (
+    <>
+      {features?.indexOf("l2-upload-report") === -1 ? 
+        <>
+          <CertificateButton />
+          <TransactionModal />
+        </> : 
         <>
-            {features?.indexOf("l2-upload-report") === -1 ? 
-                <>
-                    <CertificateButton />
-                    <TransactionModal />
-                </> : 
-                <>
-                    <LoadingButton
-                        variant="contained" size="large"
-                        className="button min-w-[200px]"
-                        onClick={() => setMetadataModalStatus(true)}
-                        loading={certifying}
-                    >
-                        <span>Review Certification Metadata</span>
-                    </LoadingButton>
-                    <MetadataModal />
-                </>
-            }
-        
-            <Snackbar
-                open={showError ? true : false}
-                autoHideDuration={5000}
-                onClose={() => setShowError("")}
-                anchorOrigin={{ vertical: 'top', horizontal: 'center' }}
-            >
-                <Alert
-                severity="error" variant="filled"
-                onClose={() => setShowError("")}
-                >
-                    {showError}
-                </Alert>
-            </Snackbar>
+          <LoadingButton
+            variant="contained" size="large"
+            className="button min-w-[200px]"
+            onClick={() => setMetadataModalStatus(true)}
+            loading={certifying}
+          >
+            <span>Review Certification Metadata</span>
+          </LoadingButton>
+          <MetadataModal />
         </>
-    );
+      }
+  
+      <Snackbar
+        open={showError ? true : false}
+        autoHideDuration={5000}
+        onClose={() => setShowError("")}
+        anchorOrigin={{ vertical: 'top', horizontal: 'center' }}
+      >
+        <Alert
+          severity="error" variant="filled"
+          onClose={() => setShowError("")}
+        >
+          {showError}
+        </Alert>
+      </Snackbar>
+    </>
+  );
 }
 
 export default CreateCertificate;
```

Comment:
Can you try to redo the changes in this file, so that only they are highlighted, rather than edits appearing in all lines here. For easiness of maintenance, and to track(`git blame`) while figuring out edits.

Reply:
Ok, I'm working on it

Reply:
Done